c2corg_api/scripts/es/sync.py

```diff
@@ -247,6 +264,24 @@ def sync_documents(session, changed_documents, batch_size):
                 create_search_documents(doc_type, docs, batch)
 
 
+def sync_deleted_documents(session, deleted_documents, batch_size):
+    client = elasticsearch_config['client']
+    batch = ElasticBatch(client, batch_size)
+    index = elasticsearch_config['index']
+    n = 0
+    with batch:
+        for document_id, doc_type in deleted_documents:
+            batch.add({
+                '_index': index,
+                '_id': document_id,
+                '_type': doc_type,
+                'id': document_id,
+                '_op_type': 'delete'
+            })
+            n += 1
+        log.info('Removed {} document(s)'.format(n))
```

Comment:
Maybe move the log statement outside the `with batch:` block. Because the (last) request to ES is only send when the batch is closed. And only then you know if it was successful.